src/views/InvolvementProfile/components/Membership/components/MemberList/components/MemberListItem/index.js

```diff
@@ -266,11 +266,17 @@ const MemberListItem = ({
             </Avatar>
           </Grid>
           <Grid item xs={3}>
-            <Link href={`/profile/${member.AD_Username}`} underline="hover">
+            {profile.PersonType?.includes?.('stu') & member.IsAlumni ? (
```

Comment:
```suggestion
            {profile.PersonType?.includes?.('stu') && member.IsAlumni ? (
```
You want the logical "AND", not the bitwise.

Reply:
> Please see my API PR comments.
> 
> Also, have you run this successfully? I think you would have to make changes to the service types since they are now written in TypeScript (.ts)

Weirdly, yes. Even though with this error, I tested it successfully. But I do need to change this to AND.